modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -39,27 +47,68 @@ EOF
 The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.15.0` to download the {microshift-short} 4.15.0 RPMs.
 ====
 
+. Optional. Use the blueprint installed in the `/usr/share/microshift/blueprint` directory that is specific to your platform architecture. See the following example snippet for an explanation of the blueprint sections:
++
+.Generated Image Builder blueprint example snippet
++
+[source,text]
+----
+name = "microshift-blueprint"
+description = "MicroShift 4.15.1 on x86_64 platform"
+version = "0.0.1"
+modules = []
+groups = []
+
+[[packages]] <1>
+name = "microshift"
+version = "4.15.1"
+...
+...
+
+[customizations.services] <2>
+enabled = ["microshift"]
+
+[customizations.firewall]
+ports = ["22:tcp", "80:tcp", "443:tcp", "5353:udp", "6443:tcp", "30000-32767:tcp", "30000-32767:udp"]
+...
+...
+
+[[containers]] <3>
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:f41e79c17e8b41f1b0a5a32c3e2dd7cd15b8274554d3f1ba12b2598a347475f4"
+
+[[containers]]
+source = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:dbc65f1fba7d92b36cf7514cd130fe83a9bd211005ddb23a8dc479e0eea645fd"
+...
+…
+EOF
+----
+<1> References for all non-optional {microshift-short} RPM packages using the same version compatible with the `microshift-release-info` RPM.
+<2> References for automatically enabling {microshift-short} on system startup and applying default networking settings.
+<3> References for all non-optional {microshift-short} container images necessary for an offline deployment.
+
 . Add the blueprint to the Image Builder by running the following command:
 +
 [source,terminal]
 ----
-$ sudo composer-cli blueprints push minimal-microshift.toml
+$ sudo composer-cli blueprints push <microshift-blueprint.toml> <1>
 ----
+<1> Replace <microshift-blueprint.toml> with the name of your TOML file.
```

Comment:
```suggestion
<1> Specify the name of your TOML file.
``

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks:~:text=%5Bsource%2Cterminal%5D%0A%2D%2D%2D%2D%0A%24%20oc%20autoscale,percent%20of%20requested%20CPU.

Reply:
same suggestion for all instances.